src/data/markdown/docs/03 cloud/05 Project and Team Management/05 SAML SSO.md

```diff
@@ -5,30 +5,36 @@ excerpt: 'Guide on setting up a SAML SSO Identity Provider with k6 Cloud'
 
 > ⭐️ SAML SSO is available on Enterprise plans.
 
-## What is SAML?
+## What is SAML SSO?
 
-Security Assertion Markup Language (SAML) is an open standard for exchanging authentication and authorization data between parties, in particular, between an Identity Provider (e.g. Azure AD, Okta) and a Service Provider (i.e. k6 Cloud). SAML SSO eases provisioning, management and access of users from a single identity provider to multiple service providers.
+[SAML SSO](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) is an open standard that allows your company to manage your users through a central Identity Provider (IdP) application (such as [Azure Active Directory](/cloud/project-and-team-management/saml-sso/azure-ad) and [Okta](/cloud/project-and-team-management/saml-sso/okta)).
+Once set up, new employees just need to be granted access in that application. They will then be able to log into k6 cloud without any additional authentication.
 
-k6 Cloud makes use of SAML single sign-on (SSO) to facilitate provisioning of new users, and giving them leveled access to various organization(s) and projects.
+Upon connecting, the user will end up in the correct k6 organization and have access to k6 [Teams](/cloud/project-and-team-management/teams) and [Projects](/cloud/project-and-team-management/projects) based on the Okta/Azure Groups you have assigned them to. Removing them from the IdP means they also completely lose access to k6.
 
-> __Note__
->
-> The SAML 2.0 standard allows k6 Cloud to work with practically any SAML 2.0 identity provider, not just the ones documented here.
+## Suggested Identity Providers (IdPs)
+
+- [Okta](https://www.okta.com) 
+- [Azure Active Directory](https://azure.microsoft.com/en-au/products/active-directory/) - . Azure is supported, but Azure only pass group IDs and not actual group names to k6 cloud. This means that for anything more advanced, you need to manually create Teams/Projects on the k6 cloud side and then contact us in order to map them to the IDs sent from Azure AD.
+- [Auth0](https://auth0.com/) is tested, but not supported, because it can't map [custom attributes](https://community.auth0.com/t/adding-custom-saml-attribute-when-auth0-is-idp/45691) in assertions.
+- Since we support the open The  [SAML 2.0](https://en.wikipedia.org/wiki/SAML_2.0) standard, you can connect k6 Cloud to practically any SAML 2.0 identity provider, not just the ones documented here.
```

Comment:
Two changes:
```suggestion
- Since we support the open [SAML 2.0](https://en.wikipedia.org/wiki/SAML_2.0) standard, you can connect k6 Cloud to practically any SAML 2.0 identity provider, not just the ones documented here. For further assistance, please contact support.
```